source/_integrations/filter.markdown

```
@@ -58,6 +58,7 @@ sensor:
         window_size: "00:05"
         precision: 2
 ```
+The example above will create two new entities: sensor.filtered_realistic_humidity and sensor.filtered_realistic_temperature 
```

Markdownlint (0.35.0)

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)